.github/workflows/syntax_check_cloud.yml

```
@@ -0,0 +1,37 @@
+# This workflow will check for exceptions to raise properly and do not generate syntax errors.
+# Also old code style using parenthesis in class definitions will be checked
+
+name: Syntax checking
+
+on:
+  [push, pull_request]
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Check exceptions
+      # Not related to tests, but to QA in general: Exceptions usually take only 1 argument
+      # So a comma is probably a sign of syntax error and should be replaced by a %
+      run: |
+        grep -IrE --colour 'raise .+".*%.*",' --include=*.py ./src/odemis ./scripts/ ./plugins/
```

> Exceptions are still failing when there are none..

Noooooo!
I guess you can change to:
```suggestion
        grep -IrE --colour 'raise .+".*%.*",' --include=*.py ./src/odemis ./scripts/ ./plugins/ || exit 0
        echo "Found the above syntax errors in exceptions, this is a sign that a comma should be replaced by a %."
        exit 1
```
(Same for the next test)